run_uts.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -x
+set -e
+date
+pwd
+git status
+
+./create_binary.sh
+docker stop pyinstaller docopt
+docker rm pyinstaller docopt
+docker run -rm -v `pwd`/:/code calico-build bash -c '/tmp/etcd & nosetests -c nose.cfg'
+
+# BE CAREFUL
```

Comment:
Interesting assertion! I quite appreciate it when people conspicuously flag things that are potentially dangerous. This is to tell people that modifications or typos might not be harmless, and to think twice instead of once.